src/FINDOLOGIC/Export/XML/XMLItem.php

```diff
@@ -169,4 +170,19 @@ private function validateImages(array $images)
 
         return $valid;
     }
+
```

Comment:
Maybe also add the getter-Methods with a `BadMethodCallException`

Reply:
I'm gonna stick with `UnsupportedValueException`, because it seems perfect for that purpose.

Reply:
:heavy_check_mark: